doc-content/enterprise-only/cases/case-management-creating-manager-approval-proc.adoc

```diff
@@ -2,88 +2,92 @@
 . Click the *Manager approval* user task and create a Data-based Exclusive (XOR) gateway.
 +
-image::cases/xor-gateway.png[Creating a Data-based Exclusive (XOR) gateway]
+image::cases/xor-gateway.png[Creating a gateway]
++
+image::cases/xor-gateway2.png[Creating a Data-based Exclusive (XOR) gateway]
+
+. Create and configure the *Place order* reusable subprocess:
+.. From the *Object Library*, expand *Subprocesses*, click *Reusable*, and drag the new element to the canvas on the right side of the Data-based Exclusive (XOR) gateway.
++
+image::cases/reusablegtwy.png[Creating a reusable task]
 
-. Click image:cases/right-arrow.png[] to open the *Object Library*, expand *Subprocesses*, click *Reusable*, and drag the new element to the canvas on the right side of the Data-based Exclusive (XOR) gateway.
-. Connect the Data-based Exclusive (XOR) gateway to the subprocess.
+.. Connect the Data-based Exclusive (XOR) gateway to the subprocess.
 +
 image::cases/connect-sub.png[Creating a Data-based Exclusive (XOR) gateway to subprocess]
 
-.  Click the subprocess, click image:cases/left-arrows.png[] to open the *Properties* panel, and input or select the following values:
-* *ActivityType* - `Sub-Process`
-* *Name* - `Place order`
-* *Called Element* - Select *place-order.bpmn* from the *Asset Name* column, and click *Save*.
-* *Independent* - *false*
-. Click the down arrow in the *Assignments* field, add the following new conditions in the *Place order Data I/O* window, and click *Save*.
+.. Click the new sub task, and in the upper-right corner, click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. Input `Place order` in the *Name* field.
+.. Expand *Data Assignments* and click image:getting-started/diagram_properties.png[] in the *Assignments* field and add the following new conditions in the *Task Data I/O* window.
 +
 image::cases/place-order-io.png[Place order I/O values]
 
-. Click the connection from the Data-based Exclusive (XOR) gateway to the subprocess and click image:cases/left-arrows.png[] to open the *Properties* panel.
-+
-image::cases/connect-sub-order.png[Exclusive gateway to subprocess]
-. Click the down arrow in the *Expression* field and add the following *Sequence Flow Conditions* and click *Ok*.
+.. Click the connection from the Data-based Exclusive (XOR) gateway to the subprocess and click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. Expand *Implementation/Execution*, select *Condition* and set the following condition expressions.
 +
 image::cases/seq-flow-conds.png[Sequence flow conditions]
 
-. Click the *Place order* user task and create an end event.
-+
-image::cases/place-order-endevent.png[Create an end event]
-
-. Click the Data-based Exclusive (XOR) gateway and create a new user task and drag the new task to align it below the *Place order* task.
+.. Click the *Place order* user task and create an end event.
+. Create and configure the *Order rejected* user task:
```

Comment:
Again if you are not referencing and UI element not bolded. The step above is correct because you say click *Place order*

Reply:
Fixed